src/main/resources/scripts/rwa-2829_mi_data_fixes.sql

```diff
@@ -0,0 +1,216 @@
+--    Results should be like below
+-- 5,674
+
+select first_assigned_date, first_assigned_date_time, wait_time_days, wait_time, role_category_label, jurisdiction_label, case_type_label,  state_label, report_refresh_time  from reportable_task rt  where jurisdiction = 'CIVIL' and state = 'ASSIGNED' and report_refresh_time is not null order by report_refresh_time desc
+--  Inspect results for non nulls
+
+select first_assigned_date, first_assigned_date_time, wait_time_days, wait_time, completed_date, completed_date_time, processing_time_days, processing_time,handling_time_days, handling_time, due_date_to_completed_diff_days, due_date_to_completed_diff_time, is_within_sla,role_category_label, jurisdiction_label, case_type_label,  state_label, final_state_label, report_refresh_time  from reportable_task rt  where jurisdiction = 'CIVIL' and state = 'TERMINATED' and termination_reason = 'completed' and report_refresh_time is not null order by report_refresh_time desc
+-- Inspect results for non nulls
+
+
+-- DATA CHANGE IMPLEMENTATION BY JURISDICTION - IA
+---------------------------------------------------------------
+-- Some dates to pick from
+select count(*) from tasks where created < to_timestamp('2022-10-20 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'IA'  -- 99,584
+select count(*) from reportable_task rt  where task_id in (select task_id  from tasks where created < to_timestamp('2022-10-20 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'IA') -- 31
+
+select count(*) from tasks where created < to_timestamp('2023-03-10 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'IA'  -- 254,232
+select count(*) from reportable_task rt  where task_id in (select task_id  from tasks where created < to_timestamp('2023-03-10 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'IA') -- 1,124
+
+select count(*) from tasks where created < to_timestamp('2023-06-20 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'IA'  -- 371,224
+select count(*) from reportable_task rt  where task_id in (select task_id  from tasks where created < to_timestamp('2023-06-20 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'IA') -- 2,961
+
+select count(*) from tasks where created < to_timestamp('2023-07-20 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'IA'  -- 412,202
+select count(*) from reportable_task rt  where task_id in (select task_id  from tasks where created < to_timestamp('2023-07-20 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'IA') -- 10,590
+
+select count(*) from tasks where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'IA'  -- 495,899
+select count(*) from reportable_task rt  where task_id in (select task_id  from tasks where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'IA') -- 83,509
+-- end of date pick up
+
+
+CALL cft_task_db.mark_report_tasks_for_refresh(null,null,'IA',null,null, '2023-09-18 16:00:00'::timestamp);
```

Comment:
Would be good to just decide on the 1,2,3 iterations and prepopulate here?  With the above we will probably have to explain what to do and copy paste a new  CALL statement.